### ISSUE TITLE
Loading a room is very slow

### ISSUE THREAD

**Issue by chagai95** [vector-im/element-android#2847]
**Describe the bug**
Loading a room is very slow

**To Reproduce**
1. Go to a room

**Expected behavior**
show last messages instantly

**Screencap**

**Smartphone (please complete the following information):**
 - Device: xiaomi 8
 - OS: MIUI 12.0.2.0

**Additional context**
 - latest App version 
 - Homeserver: chagai.website

Add any other context about the problem here:

Maybe cache more/better? Is this related?

https://github.com/vector-im/element-android/issues/1476

**Comment by bmarty** [comment]
Thanks for the comparison.
Are you using the debug version of the Element app?

**Comment by meain** [comment]
This happens on version 1.1.0 [40101000] (F-bb33a92d) and sdk version 0.0.1 (bb33a92d). Clearing the cache seems to help a bit though.

**Comment by BloodyIron** [comment]
Yeah I'm still finding that Element loading (in the background, updating rooms/dms/whatever) or even going into a room the first time, is painfully slow. (Android in this context).

Element should be updating all these things for me automatically in the background and when I go to open it, I shouldn't have to wait even more for it to scan for updates...  when this is already serialised data?
